README.md

```diff
@@ -1,12 +1,27 @@
-# bpmn-visualization examples
-
-[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](https://github.com/process-analytics/.github/blob/main/CODE_OF_CONDUCT.md)
-[![License](https://img.shields.io/github/license/process-analytics/bpmn-visualization-examples?color=blue)](LICENSE)
+<h1 align="center">BPMN Visualization Examples</h1> <br>
+<p align="center">
+    <p align="center">
+        <a href="https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/demo/index.html">
+            <img src="examples_home.png">
```

Comment:
I still disagree with the content of this image. We could replace it with the hacktoberfest image, see #63

Reply:
It's possible for the Hacktoberfest.
But in my opinion, the image of this repository is more the screenshot of the home of all examples (as this repository regroup them) than a screenshot of the library.
It can be confusing for the user to have the same image, and to have a image who don't represent the content of the repo 😄